docs/tool.md

```diff
@@ -0,0 +1,111 @@
+If you want to use `homebase-lite` on MacOS, you can try [building it from source](#build-instructions) using `stack`.
+
+### Executables
+
+In order to use the `homebase-lite` tool or run network tests, you will need to obtain the `tezos-client` executable, which is used for key storing and operation signing.
+
+Various forms of distribution for `tezos-client` are presented in the [tezos-packaging repo](https://github.com/serokell/tezos-packaging).
+
+## How to get
+
+You can obtain `homebase-lite` tool either by building from source, or downloading an automatically built static executable from [the latest release](https://github.com/tezos-commons/homebase-lite/releases/latest).
+
+### Build instructions
+
+This project is using the [Haskell Tool Stack](https://docs.haskellstack.org/en/stable/install_and_upgrade/), please follow the link to find instructions on obtaining the `stack` tool.
+
+Once you have `stack`, building should be as easy as cloning the repository and running `stack install` in the root of the cloned tree:
+
+```bash
+$ git clone https://github.com/tezos-commons/homebase-lite.git
+$ cd homebase-lite
+$ stack install
+```
+
+## Usage
+
+Once you've obtained the `homebase-lite` executable,
+use `homebase-lite --help` to get a list of available commands.
+
+You can also get help for a specific command by running `homebase-lite COMMAND --help`.
+
+### Getting the Michelson code of the compiled contract
+
+Use `homebase-lite print` to get the Michelson code of the contract.
+
+This will save the contract code to `homebase-lite.tz` in the current working directory. Alternatively, add `--output` option to change the output path, or use `-` to print to standard output:
+
+```bash
+$ homebase-lite print --output -
+```
+
+### Getting the initial storage for the contract
+
+Use `homebase-lite storage` to get the initial storage for the contract.
+
+You'll need to at least specify the admin keyhash using the `--admin` key and the address of the FA2 contract that will be used to check for governance token balance with `--fa2-address` option, for example:
+
+```bash
+$ homebase-lite storage --admin tz1Znvao83anTh654Uu44Kb9atTp5yTK93A8 --fa2-address KT1AbgvM5D1wAVZbsB3WTCB54E94p2hn1Rb9
+```
+
+You can also specify other initial values. Consult `homebase-lite storage --help` for more information.
+
+### Getting the contract documentation
+
+Use `homebase-lite document`. That will produce `homebase-lite.md` file in the current working directory. This documentation is also attached to binary releases.
+
+You can use `--output` option to specify a different output path, or use `-` to print to standard output.
+
+### Deploying the contract
+
+You could print the contract's Michelson code and initial storage and then just use `tezos-client originate` if you wanted to, but the `homebase-lite` tool provides a more convenient command: `homebase-lite originate`. You will need to have the `tezos-client` executable available on your system.
```

Comment:
Likewise this could be:
```suggestion
You could print the contract's Michelson code and initial storage and then just use `tezos-client originate` if you wanted to, but the `homebase-lite` tool provides a more convenient command: `homebase-lite originate`. You will need to have `tezos-client` available on your system.
```
but both are ok to me, so feel free to ignore this :slightly_smiling_face: